client_test.go

```diff
@@ -16,6 +16,7 @@ func TestPacketJSON(t *testing.T) {
 	packet := &Packet{
 		Project:    "1",
 		EventID:    "2",
+                Release:    "rel1",
```

Comment:
The [Guide](This value will generally be something along the lines of the git SHA for the given project.) says of the `release` attribute:

> This value will generally be something along the lines of the git SHA for the given project.
>
> ```
> {
>     "release": "721e41770371db95eee98ca2707686226b993eda"
> }
> ```

So it would make sense to use that kind of value here.